### ISSUE TITLE
Exception when launching Sublime Text 3 on Fedora 20

### ISSUE THREAD

**Issue by allanlewis** [codexns/sublime-ssl-linux#1]
When I launch Sublime I'm getting this error:

```
reloading plugin 0_package_control_loader.00-package_control
reloading plugin 0_package_control_loader.01-ssl-linux
Traceback (most recent call last):
  File "01-ssl-linux in /home/alewis/.config/sublime-text-3/Installed Packages/0_package_control_loader.sublime-package", line 22, in <module>
ImportError: libssl.so.1.0.0: cannot open shared object file: No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/sublime_text/sublime_plugin.py", line 73, in reload_plugin
    m = importlib.import_module(modulename)
  File "./importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "/opt/sublime_text/sublime_plugin.py", line 671, in load_module
    exec(compile(source, source_path, 'exec'), mod.__dict__)
  File "01-ssl-linux in /home/alewis/.config/sublime-text-3/Installed Packages/0_package_control_loader.sublime-package", line 26, in <module>
NameError: name 'e' is not defined
```

This started happening today - I'm not sure why. I have my configuration synchronised with Dropbox as per the instructions on http://wbond.net/ - I did that months ago and I haven't had any major issues. My installed `libssl` is `1:1.0.1e-40.fc20`, but that's been the case since October '14.

I tried sym-linking from `/usr/lib64/libssl.so.1.0.0` to `/usr/lib64/libssl.so.1.0.1e`: that gave me a similar error about `libcrypto.so.1.0.0`. Sym-linking that left me with an error about the internals of that library; that's where I gave up.

**Comment by wbond** [comment]
This should be fixed by version 1.0.1, which is bootstrapped by PC 3.0.1.

**Comment by allanlewis** [comment]
Thanks @wbond.
